.github/workflows/checks.yml

```diff
@@ -118,34 +119,47 @@ jobs:
           path: .security.json
           include-hidden-files: true
 
+  Format:
+    name: Format Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
+        with:
+          python-version: "3.9"
+
+      - name: Run format check
+        run: poetry run nox -s project:format
+
   Tests:
-    name: Tests (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
-    needs: [ Documentation, Lint, Type-Check, Security ]
+    name: Unit-Tests (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
+    needs: [ Documentation, Lint, Type-Check, Security, Format, build-matrix ]
```

Comment:
AFAIK the job is called "Format Check" rather than "Format".
Maybe we should rename the job to "Format-Check" to not contain a space character?

Reply:
Additionally I propose to rename "build-matrix" to either Build-Matrix or Create-Build-Matrix.

Reply:
"Format" is the name of the job. Only the job's first step is name "Format Check".
Spelling of "matrix-build" remains unchanged for now and is inherited from PTB.